### DISCUSSION TITLE
Yocto Scarthgap Transition ?

### DISCUSSION THREAD

**Discussion by lucas-hode** [polarfire-soc/polarfire-soc-documentation#450] [Linux build systems for PolarFire SoC] [answered]
Hey, I was wondering if the next releases of the meta-polarfire-soc-yocto-bsp would still use the kirkstone release of yocto or if you were planning on using the new scarthgap release ?
Thanks in advance !

**Comment by vfalanis** [comment, marked as answer]
Hi @lucas-hode ,

The next release of the meta-polarfire-soc-yocto bsp is planned to update to a more recent kirkstone version.

Thanks

**Reply by lucas-hode** [reply]
Alright, thanks for the answer :)
